dht-node/src/dht_node/index.ts

```diff
@@ -203,7 +218,56 @@ async function writeFederatedHomeCommunityEnries(pubKey: any): Promise<Community
       logger.info(`federation home-community inserted successfully: ${JSON.stringify(homeCom)}`)
     })
   } catch (err) {
-    throw new Error(`Federation: Error writing HomeCommunity-Entries: ${err}`)
+    throw new Error(`Federation: Error writing federated HomeCommunity-Entries: ${err}`)
   }
   return homeApiVersions
 }
+
+async function writeHomeCommunityEntry(pubKey: any): Promise<void> {
```

Comment:
this is unreleated to the rest of the PR.
With the sharing of secrets between the services with .env files our initial commitment to use the database to share values between the services is obsolete and therefore the writing of the home community to the database is in question for both tables.

Reply:
writing the homeCommunity data in database is one of the conceptual topics you all aggreed.
This is totally indepentent from where we store generated secrets.

Here you can see an example, where i understand your own arguements in the very reverse:
1. our initial commitment to use the database to share values between the services
2. no the generated private key must not be stored in database